src/main/java/nextstep/auth/JwtTokenProvider.java

```diff
@@ -31,6 +30,10 @@ public String getPrincipal(String token) {
         return Jwts.parser().setSigningKey(secretKey).parseClaimsJws(token).getBody().getSubject();
     }
 
+    public String getRole(String token) {
+        return Jwts.parser().setSigningKey(secretKey).parseClaimsJws(token).getBody().get(AUTHORITIES_KEY).toString();
+    }
```

Comment:
> 2 . 이런 문제를 어떻게 하면 더 잘 해결할 수 있을까요?

캐스팅에 대한 질문을 주신 것 같은데요! 
Enum에 대한 타입 캐스팅이라면 아래와 같이 변경해보는건 어떨까요? :)

```suggestion
    public MemberRole getRole(String token) {
        String s = Jwts.parser()
                .setSigningKey(secretKey)
                .parseClaimsJws(token)
                .getBody()
                .get(AUTHORITIES_KEY)
                .toString();
        
        return MemberRole.valueOf(s);
    }
```